### PR TITLE
Add conference data to the Database Project.

### PR DESCRIPTION
**Pull request by alastairs**
Again, this is added to the post-deployment script. There's probably a better way to add this kind of data, but I've not found it yet. 

This should make a cleanly-deployed database project work out of the box.

### CODE REVIEW

File: DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql [hunk 1]
```diff
@@ -96,4 +96,130 @@ INSERT INTO [dbo].[__MigrationHistory] (
...
+    [EndDate],
+    [EntryType],
+    [EntryTypeString]
+) VALUES (
+    3,
+    1,
+    1,
+    1,
+    'Agenda announced',
+    '2013-05-01 00:00:00.0000000 +01:00',
```
Comment (adrianbanks) [inline DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql]: The agenda is announced on 29th May, so this should be `2013-05-29 00:00:00.....`.

File: DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql [hunk 2]
```diff
@@ -96,4 +96,130 @@ INSERT INTO [dbo].[__MigrationHistory] (
...
+    [EndDate],
+    [EntryType],
+    [EntryTypeString]
+) VALUES (
+    4,
+    1,
+    1,
+    1,
+    'Registration',
+    '2013-06-01 00:00:00.0000000 +01:00',
```
Comment (adrianbanks) [inline DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql]: Registration opens on 3rd June.

File: DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql [hunk 3]
```diff
@@ -96,4 +96,130 @@ INSERT INTO [dbo].[__MigrationHistory] (
...
+    [EntryType],
+    [EntryTypeString]
+) VALUES (
+    5,
+    1,
+    1,
+    1,
+    'Conference',
+    '2013-06-29 08:30:00.0000000 +01:00',
+    '2013-06-29 18:30:00.0000000 +01:00',
```
Comment (adrianbanks) [inline DDDEastAnglia.Database/Scripts/Post-Deployment/Script.PostDeployment.sql]: The conference ends at 17:00.

**Comment by adrianbanks** [comment]
Why do all of the times have `+01:00` on them?

**Comment by alastairs** [comment]
Time zone difference: Azure operates on UTC, the dates/times of the milestones are all in BST which is one hour ahead of UTC.

**Comment by alastairs** [comment]
I'll update the pull request to fix the dates/times as described. These were pulled from (i.e. copy-pasted) from the Azure database, although I forget now whether it was production or staging, so they will need adjusting there too.

**Comment by alastairs** [comment]
Updated as requested.

**Pull request merged**